builtin/builtin.mbti

```diff
@@ -106,6 +106,7 @@ impl Array {
   rev_inplace[T](Self[T]) -> Unit
   search[T : Eq](Self[T], T) -> Int?
   search_by[T](Self[T], (T) -> Bool) -> Int?
+  set[T](Self[T], Int, Self[T], Int, Int) -> Unit
```

Comment:
why this one is needed?

Reply:
It's not necessary, it just retains a function similar to JavaScript's array, which can also be removed